turbinia/task_manager.py

```diff
@@ -101,12 +101,24 @@ def _backend_setup(self, *args, **kwargs):
     """
     raise NotImplementedError
 
-  def setup(self, *args, **kwargs):
-    """Does setup of Task manager and its dependencies."""
+  def setup(self, jobs_blacklist=None, jobs_whitelist=None, *args, **kwargs):
+    """Does setup of Task manager and its dependencies.
+
+    Args:
+      jobs_blacklist (list): Jobs that will be excluded from running
```

Comment:
(Optional[list[str]), not (list)

Reply:
https://github.com/log2timeline/l2tdocs/blob/master/process/Style-guide.md#docstrings

Reply:
Thanks!